source/_integrations/lg_thinq.markdown

```diff
@@ -26,12 +26,12 @@ ha_codeowners:
 
 The **LG ThinQ** integration allows you to connect LG ThinQ devices to Home Assistant. The features of this integration include:
 
-- Control LG appliances as Home Assistant entities through the [LG ThinQ Connect API](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/overview/).
+- Control LG appliances as Home Assistant entities through the [LG ThinQ Connect API](https://github.com/thinq-connect/pythinqconnect).
```

Comment:
I think we can remove this point altogether. For the same reason as raised earlier, the end user doesn't have to know internal implementation details or need links to source code of internally used package.

../Frenck